src/bunit.web.query/Labels/ByLabelTextOptions.cs

```diff
@@ -0,0 +1,17 @@
+namespace Bunit;
+
+/// <summary>
+/// Allows overrides of behavior for FindByLabelText method
+/// </summary>
+public record ByLabelTextOptions
+{
+	/// <summary>
+	/// The default behavior used by FindByLabelText if no overrides are specified
+	/// </summary>
+	public static readonly ByLabelTextOptions Default = new();
+
+	/// <summary>
+	/// The StringComparison used for comparing the desired Label Text to the resulting HTML. Defaults to Ordinal (case sensitive).
+	/// </summary>
+	public StringComparison ComparisonType { get; set; } = StringComparison.Ordinal;
```

Comment:
```suggestion
	public StringComparison ComparisonType { get; init; } = StringComparison.Ordinal;
```

We should not allow this to be changed as options are reused to by the wrapper factories to reissue queries after renders, and if the value can be changed, users could inadvertently change the behavior query by changing this, and it would not be obvious to them.

Reply:
They could also accidentally change the `Default` object.

Reply:
Egil fixed this in [6c646b9](https://github.com/bUnit-dev/bUnit/pull/1252/commits/6c646b9816de65a7406dffa0e854a72b53f75b7d)